backend/server.py

```
@@ -506,6 +619,28 @@ def upload_file():
+        "async_enabled": os.getenv('USE_ASYNC', 'false').lower() in ('true', '1', 'yes'),
+        "services": {
+            "flask": "running"
+        }
+    }
+    
+    if USE_CELERY_INFERENCE:
+        # Check if Celery is accessible
+        try:
+            from inference_service import CELERY_AVAILABLE
+            health_status["services"]["celery"] = "available" if CELERY_AVAILABLE else "unavailable"
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)